test/runtests.jl

```diff
@@ -243,8 +238,8 @@ end
 
     # Check with noise
     X = sol[:, :] + 1e-3*randn(size(sol[:,:])...)
-    set_threshold!(opt, 3.5e-1)
-    Ψ = SInDy(X, DX, basis, maxiter = 10000, opt = opt, denoise = true, normalize = true)
+    #set_threshold!(opt, 3.5e-1)
+    Ψ = SInDy(X, DX, basis, λs, maxiter = 10000, opt = opt, denoise = true, normalize = true)
```

Comment:
is `sparse_regression` going to be considered user-level API? If so, the user-side usage should probably get a test.

Reply:
Yes, working on this. Also to be consistent add the same for implicit sindy.